### DISCUSSION TITLE
disabling toc on one page

### DISCUSSION THREAD

**Discussion by sheldygg** [squidfunk/mkdocs-material#6247] [Q&A] [answered]
In start of file i put this
```
---
hide:
  - toc
---
```
TOC is indeed hidden on this page, but not on the simplified page =)

![image](https://github.com/squidfunk/mkdocs-material/assets/85823514/9a929db8-3b3e-4bb0-92d6-f706043214a4)
![image](https://github.com/squidfunk/mkdocs-material/assets/85823514/80272d17-792a-41c5-bab1-bef076b60071)

**Comment by alexvoss** [comment]
I tested this with a sample project and it works for me. In fact, without the TOC hidden, I never see this icon used when using a mobile device? Could be other settings you have in your `mkdocs.yml`. Can you produce a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/)? That might help you to pinpoint what is going on and if you post it here we can have a look.

**Reply by sheldygg** [reply]
`mkdocs.yml`
```yml
site_name: My Docs

theme:
  name: material
```
`docs/index.md`
```md
---
hide:
  - toc
---

# Welcome to MkDocs

For full documentation visit [mkdocs.org](https://www.mkdocs.org).

## Commands

* `mkdocs new [dir-name]` - Create a new project.
* `mkdocs serve` - Start the live-reloading docs server.
* `mkdocs build` - Build the documentation site.
* `mkdocs -h` - Print help message and exit.

## Project layout

    mkdocs.yml    # The configuration file.
    docs/
        index.md  # The documentation homepage.
        ...       # Other markdown pages, images and other files.

```

![image](https://github.com/squidfunk/mkdocs-material/assets/85823514/5944d7a9-4ddf-43fc-a655-ab2dd85e5cf1)
![image](https://github.com/squidfunk/mkdocs-material/assets/85823514/9aba0166-e234-4fb3-8304-89f3d99f9b4a)
![image](https://github.com/squidfunk/mkdocs-material/assets/85823514/08739a91-8aad-449d-9b0c-22be34bad84d)

**Reply by alexvoss** [reply]
Can you please go through the process of producing the zip file as described in the page linked above so we can also replicate your environment?

**Reply by sheldygg** [reply]
yes, sorry
[9.4.6-toc-doesnt-disabled-in-simplify-version.zip](https://github.com/squidfunk/mkdocs-material/files/13175242/9.4.6-toc-doesnt-disabled-in-simplify-version.zip)

**Reply by alexvoss** [reply]
The menu you seem to be getting seems to be the top-level navigation, which it seems gets auto-populated with the toc when there is no navigation defined and only one page? That looks like an intended functionality to me and it is not created by Material but rather by MkDocs - as you can see when you comment out the theme in `mkdocs.yml`.

**Reply by sheldygg** [reply]
Does that mean I can't do anything about it? either turn off TOS completely or leave it like that?

**Reply by alexvoss** [reply, marked as answer]
You can always [customize the theme](https://squidfunk.github.io/mkdocs-material/customization/) by creating a new template for the page(s) in question.

**Comment by squidfunk** [comment]
Duplicate of #5948 😉
